src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1344,13 +1345,22 @@ private bool OnlyWhitespaceOrCommentsAfterExtent(InternalScriptExtent extent)
 
         internal bool IsPipeContinuance(IScriptExtent extent)
         {
-            return extent.EndOffset < _script.Length && PipeContinuanceAfterExtent(extent);
+            // If the first non-whitespace & non-comment (regular or block) character following a newline is a pipe, we have
+            // pipe continuance.
+            return extent.EndOffset < _script.Length &&
+                   ContinuanceAfterExtent(extent, continuanceChar: '|', skipComment: true);
         }
 
-        private bool PipeContinuanceAfterExtent(IScriptExtent extent)
+        internal bool IsTernaryContinuance(IScriptExtent extent)
```

Comment:
I would personally prefer this as `continuation`. Try searching for `line continuance` vs `line continuation` on the internet. We already use the term `continuation` in `ScanLineContinuation()` and `TokenKind.LineContinuation` as well. I believe `continuance` is more of a legal procedural term.

Reply:
Agreed. Will update.